frontend/src/components/NotificationsCentral/style.scss

```diff
@@ -0,0 +1,42 @@
+@use '../../theme/variables.scss' as vars;
+
+.#{vars.$bcgov-prefix}--row{
```

Comment:
Add a custom class to the Row component tag, something like `className="notifications-button-row"`, and use that class name to apply styles. Using directly `.#{vars.$bcgov-prefix}--row` will make the changes globally to all rows.